common/test/acceptance/pages/studio/settings_advanced.py

```diff
@@ -34,6 +37,40 @@ def _is_finished_loading():
         EmptyPromise(_is_finished_loading, 'Finished rendering the advanced policy items.').fulfill()
         return self.q(css='body.advanced').present
 
+    def get_key_names(self):
+        """
+            Returns a list of key names of all settings.
+        """
+        return self.q(css=KEY_CSS).text
+
+    def get_deprecated_settings_button_text(self):
+        """
+            Returns text for deprecated settings button
+        """
+        return self.q(css=DEPRECATED_SETTINGS_BUTTON_SELECTOR).text[0]
+
+    def click_deprecated_settings_button(self):
+        """
+            Show deprecated Settings
+        """
+        self.q(css=DEPRECATED_SETTINGS_BUTTON_SELECTOR).click()
+        if self.get_deprecated_settings_button_text() == 'Show Deprecated Settings':
+            self.wait_for_element_presence(DEPRECATED_SETTINGS_SELECTOR, 'Deprecated Settings are not present')
+        else:
+            self.wait_for_element_absence(DEPRECATED_SETTINGS_SELECTOR, 'Deprecated Settings are present')
+
+    def check_deprecated_settings_presence(self):
```

Comment:
Nit: I like that this is following the guidelines of keeping browser interaction in the page object (https://bok-choy.readthedocs.io/en/latest/guidelines.html#put-browser-interactions-in-the-page-object-not-the-test), but I think that the function name could be something like `is_deprecated_settings_visible` to make that more clear. `check` sounds like an action, rather than a state.

Reply:
The method: check_deprecated_settings_presence does not belong to this PR, I will change the name of the method in next PR, if required.

Reply:
@aghaawais01 
Since this does not belong to the current PR, please remove the method. You can add it afterwards with the suggested change, same goes with the similar case discussed below